tests/tck/features/match/WithUnwind.feature

```diff
@@ -37,6 +37,54 @@ Feature: With clause and Unwind clause
       """
     Then a SemanticError should be raised at runtime: Expression in WITH must be aliased (use AS)
 
+  Scenario: with agg return
```

Comment:
You could add some `nested aggregate function` test cases, such as `with avg(v.age)+1 AS age`.

Reply:
The nested situation is included in 
```
      """
      MATCH (v :player{name:"Tim Duncan"})-[]-(v2)
      WITH avg(v2.age) as average_age
      RETURN average_age
      """
```